docs/custom_ruleset.md

```diff
@@ -47,3 +47,79 @@ You can now click `Run Analysis` and your custom ruleset will be used.
 
 > [!WARNING]
 > You will need to make sure that the rules or rulesets have a target or source label that is selected. If the ruleset does not match a selected target or source, it will be filtered out and skipped.
+
+## Creating Rules and Rulesets
+
+### Creating Rules
+
+To create your own rules, first you need to know of a change that must happen to complete a migration. This could be because of a framework or library that you are using (either an internal one or external/open source one). For this section, I am going to create a rule and ruleset for an open source framework but the process should be similar for any custom rule.
```

Comment:
```suggestion
To create your own rules, first, you need to know of a change that must happen to complete a migration. This could be because of a framework or library that you are using (either an internal one or external/open source one). For this section, I am going to create a rule and ruleset for an open source framework but the process should be similar for any custom rule.
```